scripts/testy_launch.py

```diff
@@ -117,140 +117,155 @@ def snapshot_exists(snapshot_id):
         raise Exit(result.stderr)
     return True if result.stdout.strip() else False
 
-def wait_instance_running(instance_id):
-    print(f"Waiting for the EC2 instance '{instance_id}' to be running ...")
-    # Code 16 corresponds to "running".
-    expected_state = "16"
-    max_retries = 20
-    num_retry = 0
+def wait_on_status_check(instance_id):
+    max_retries = 30
```

Comment:
It seems that it is not enough:
```
Creating a testy server from snapshot 'snap-0049c2cd076e6227c' ... Success!
Waiting for the status check to complete. This may take several minutes ... Timed out.
Failed.
Launch failed. The status check failed to complete successfully after 300 seconds. Please check the AWS console.
```
Let's make it twice more?

Reply:
Updated retries to 60 in [8697bd9](https://github.com/wiredtiger/testy/pull/50/commits/8697bd96b441d535debde673f36804ec72214c5e).